overlord/ifacestate/udevmonitor/udevmon.go

```diff
@@ -50,13 +49,21 @@ type Monitor struct {
 	monitorStop   chan struct{}
 	netlinkErrors chan error
 	netlinkEvents chan netlink.UEvent
+
+	// seen keeps track of all observed devices to know
```

Comment:
Something off about the format of this comment, can you `M-q`?

Reply:
Done. It's probably because I split it manually.